.github/workflows/rpmbuild.yml

```diff
@@ -10,11 +10,25 @@ jobs:
       - name: checkout cli
         uses: actions/checkout@v2
 
-      - name: get version
+      - name: fetch version
         id: get-tag
         run: |
           echo "::set-output name=TAG_NAME::$(node -p -e "require('./package.json').version")"
-
+          
+      - name: process version and release
+        if: contains(steps.get-tag.outputs.TAG_NAME, 'rc' )
+        id: process-release-version
+        run: |
+          releaseVersionString=${{ steps.get-tag.outputs.TAG_NAME}}
```

Comment:
How will the workflow release, say 1.1.23-rc.1 version? I didn't understand at which step the version rc.0, rc.1, rc.2... these are interpreted ?

Reply:
- That step has to be implemented in rc release workflow in twilio cli
- rpm build just read the version released in cli and create required files and upload
- version increment and update is not part of rpmbuild